modules/ROOT/pages/_partials/proc_self-signed-certs.adoc

```diff
@@ -29,26 +28,25 @@ base64 --decode | sed -e '1,/^-----END RSA PRIVATE KEY-----$/ d'  > /tmp/localcl
 The resulting file is located in the `/tmp` directory.
 
 [[installing-on-device]]
-== Installing the OpenShift Root Certificate Authority Cert on a Device
+== 2. Installing the OpenShift Root Certificate Authority Cert on a Device
+
+[role="primary"]
+.Android
 
-. Configure a pin to ensure the certificate can be installed.
+****
+.. Enable PIN protection on the mobile device to ensure the certificate can be installed.
```

Comment:
Is there a link to a android doc we could add here which gives instructions on how to do this?